roles/default/kiali-deploy/tasks/main.yml

```diff
@@ -285,11 +285,22 @@
   when:
     kiali_vars.external_services.tracing.in_cluster_url == "" and is_maistra
 
-- name: Set default Tracing in_cluster_url
+- name: Set default Tracing in_cluster_url for grpc consumption
+  set_fact:
+    kiali_vars: "{{ kiali_vars | combine({'external_services': {'tracing': {'in_cluster_url': 'http://tracing.' + kiali_vars.istio_namespace + ':16685/jaeger'}}}, recursive=True) }}"
```

Comment:
Just pointing out this is 16685 (when deployed with Maistra it is 16686 -- see above line 284) - just want to make sure this isn't a typo and you meant it to end with "5".

Reply:
Thanks for the double check. The new port for gRPC is the `16685` while in older versions the opened port was 16686 for both HTTP and gRPC. In new versions, they moved all the traffic to port 80 (which is the version we currently support in master). And now it is splitted: 80:16686 for HTTP and 16685:16685 for gRPC.

```
$ k get svc -n istio-system tracing -o yaml | more
...
spec:
  clusterIP: 10.103.72.232
  clusterIPs:
  - 10.103.72.232
  ports:
  - name: http-query
    port: 80
    protocol: TCP
    targetPort: 16686
  - name: grpc-query
    port: 16685
    protocol: TCP
    targetPort: 16685

``

Reply:
Never mind - I see in the jaeger PR that the port is 16685: https://github.com/jaegertracing/jaeger/pull/2387/files#diff-872d5d3fba389504d376a8dc71533d58db75d6e4bcbceaace2cf383d8352a62aR42

Reply:
Note that this is the pod port, not the service one. Therefore, we should rely more on this one: https://github.com/istio/istio/blob/master/samples/addons/jaeger.yaml#L72
However, as usual, we really rely on user's deployment and this might change.